jellyfin_kodi/helper/utils.py

```diff
@@ -214,8 +214,8 @@ def set_screensaver(value):
 
 class JSONRPC(object):
 
-    version = 1
-    jsonrpc = "2.0"
+    id = 1
+    jsonrpc_version = "2.0"
```

Comment:
I remember looking at this change myself, I do not remember why I didn't.

Reply:
Below is the only place I could find which referenced these variables and everything in my testing still worked so I'd assume it's all okay